src/components/Board.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div v-if="gameStats.gameStart">
     <h2>{{ msg }}</h2>
-    
+    <div class="wrapper">
+      <div v-for="(row, index) in gameSquares" :key="index" class="row">
+        <p>{{ index }}</p>
+        <div v-for="square in row" :key="square.id" :class="square.classes">
+          <p>{{ square.id }}</p>
```

Comment:
here I am outputting the square ID for each square in the board just to see it in the page. This can be deleted later.